bundles/auth/core/src/main/java/org/apache/sling/auth/core/impl/SlingAuthenticator.java

```diff
@@ -1414,7 +1414,7 @@ private void redirectAfterLogout(final HttpServletRequest request,
 
         // redirect to there
         try {
-            response.sendRedirect(request.getContextPath() + target);
+            response.sendRedirect(target);
```

Comment:
This at least is not correct in case the redirect is not valid (then target is "/" and this is obviously lacking the context path).